components/homme/src/theta-l_kokkos/cxx/cxx_f90_interface_theta.cpp

```diff
@@ -62,7 +62,14 @@ void init_simulation_params_c (const int& remap_alg, const int& limiter_option,
   Errors::check_option("init_simulation_params_c","nu",nu,0.0,Errors::ComparisonOp::GT);
   Errors::check_option("init_simulation_params_c","nu_div",nu_div,0.0,Errors::ComparisonOp::GT);
   Errors::check_option("init_simulation_params_c","theta_advection_form",theta_adv_form,{0,1});
+#ifndef SCREAM
   Errors::check_option("init_simulation_params_c","nsplit",nsplit,1,Errors::ComparisonOp::GE);
+#else
+  if (nsplit<1) {
```

Comment:
In SCREAM, we do not have access to dt when Homme is inited. However, nsplit is only used at run time, so here I'm just issuing a warning if nsplit<1. In `prim_run_subcycle` (from theta-l_kokkos folder), we start by checking that nsplit has indeed been set to a positive value (leaving it up to the user to make sure the timestep parameters are consistent).